ecl/hql/hqlgram2.cpp

```diff
@@ -10940,7 +10942,7 @@ static void getTokenText(StringBuffer & msg, int token)
     }
 }
 
-inline bool containsToken(int first, const int * expected)
+static inline bool containsToken(int first, const int * expected)
```

Comment:
no problem with the change, I'm not sure it should be inline in the first place.